documentation/modules/managing/proc-pausing-reconciliation.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// assembly-management-tasks.adoc
+
+[id='proc-pausing-reconciliation-{context}']
+
+= Pausing reconciliation of custom resources
+
+Sometimes it is useful to pause the reconciliation of custom resources managed by Strimzi Operators,
+so that you can perform fixes or make updates.
+If reconciliations are paused, any changes to custom resources are ignored by Strimzi Operators until the pause ends.
+
+If you want to pause reconciliation of a custom resource, set the `pause-reconciliation` annotation to `true` in its configuration.
+This instructs the appropriate Operator to pause reconciliation of the custom resource.
+For example, you can apply the annotation to the `KafkaConnect` resource so that reconciliation by the Cluster Operator is paused.
+
+You can also _create_ a custom resource with the pause annotation enabled.
+The custom resource is created, but it is ignored.
+
+To resume reconciliation, you can set the annotation to `false`, or remove the annotation.
+
+IMPORTANT: It is not currently possible to pause reconciliation of `KafkaTopic` resources.
+
+.Prerequisites
+
+* The Strimzi Operator that manages the custom resource is running.
+
+.Procedure
+
+. Annotate the custom resource in Kubernetes, setting `pause-reconciliation` to `true`:
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate _KIND-OF-CUSTOM-RESOURCE_ _NAME-OF-CUSTOM-RESOURCE_ strimzi.io/pause-reconciliation: "true"
+----
++
+For example, for the `KafkaConnect` custom resource:
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate KafkaConnect my-connect strimzi.io/pause-reconciliation: "true"
+----
+
+. Check that the status of the custom resource has changed to `ReconciliationPaused`:
```

Comment:
I think we should clarify that they should look for this in the `conditions` section of the status. Maybe Standa can provide an example YAML of the paused resource?

Reply:
out of curiosity, @scholzj is that an additional condition, or when the operator is paused you see the `ReconciliationPaused` condition type but there is no more `Ready` anymore?

Reply:
Yeah. I moved it out of the procedure, but I think maybe as a next step works best

![image](https://user-images.githubusercontent.com/47596553/110928664-0d949c00-831f-11eb-9faa-c27efa2b5d8a.png)